src/cryptography/hazmat/backends/openssl/cmac.py

```diff
@@ -65,7 +65,7 @@ def finalize(self):
 
         return self._backend._ffi.buffer(buf)[:]
 
-    def copy(self):
+    def copy(self) -> "_CMACContext":
```

Comment:
Should this be the interface, not the concrete type?

Reply:
There's no CMAC interface 😄. For HMAC and Hash we also have a structure where we instantiate an object which in turn instantiates a context. Every user-facing operation goes through the primary object, which in turn invokes the underlying context. So the user will never actually interact with these context objects directly.